be/src/util/simd/bits.h

```diff
@@ -57,5 +57,35 @@ inline uint32_t bytes32_mask_to_bits32_mask(const bool* data) {
     return bytes32_mask_to_bits32_mask(reinterpret_cast<const uint8_t*>(data));
 }
 
+inline size_t count_zero_num(const int8_t* data, size_t size) {
+    size_t num = 0;
+    const int8_t* end = data + size;
+
+#if defined(__SSE2__) && defined(__POPCNT__)
```

Comment:
rethink we need to do simd by ourself ? the code can auto simd by compiler and have `AVX2` 
```
    for (; data < end; ++data) {
        num += (*data == 0);
    }
```

Reply:
delete the code